tokenserver-db/src/models.rs

```diff
@@ -26,6 +26,11 @@ const MAX_GENERATION: i64 = i64::MAX;
 
 type Conn = PooledConnection<ConnectionManager<MysqlConnection>>;
 
+#[cfg(not(debug_assertions))]
+type InternalConn = Conn;
+#[cfg(debug_assertions)]
+type InternalConn = LoggingConnection<Conn>; // display SQL when RUST_LOG="diesel_logger=trace"
```

Comment:
we might wanna document this somewhere as it could be very useful when debugging.

Reply:
It's unfortunately going away with the move to `diesel-async`